app/presenters/claim/advocate_interim_claim_presenter.rb

```diff
@@ -10,4 +10,9 @@ def type_identifier
   def can_have_disbursements?
     false
   end
+
+  def raw_warrant_fees_total
+    claim.warrant_fee&.amount || 0
```

Comment:
This doesn't seem to have test coverage (according to code climate)

Reply:
⏩  🔬